app/controllers/users_controller.rb

```diff
@@ -86,7 +86,7 @@ def set_vars_from_curr_user
 	    @height = current_user.height * 2.54
 	    @exercise = current_user.exercise
 	    @goal = current_user.goal
-	    @budget = current_user.budget
```

Comment:
Looks like you removed everything pretty comprehensively to me!

Reply:
I actually didn't.. good looks